### PR TITLE
JS TODO List

### PR DESCRIPTION
**Pull request by RokoVarano**
This is a simple list for creating projects and creating events in it to keep track of them.

This project currently features:
- A project form to create projects
- A project list, which displays current projects
- A task form, which displays when a project name is clicked
- A task list, which displays tasks created with their date.
- Every element in either list has a delete button.

### CODE REVIEW

**Review by MkrtichSargsyan — CHANGES_REQUESTED** [review]
## _Status:_ Changes Requested :recycle:

## Hello Emmisteel Okello and Rodrigo Ibaceta Jimenez

THIS IS VERY IMPRESSIVE WORK. YOU HAVE DONE A GOOD JOB :+1:

## To Highlight :tada:

1. The project is almost done.
3. Your PR has a good title and description.
4. You have descriptive git commit messages.

However, there are some changes that you need to make in order to get your project approved

- [x] Don't disable eslint rules

- [x] DOM-Manipulation and Logic are in separate modules [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/modules_requirement.png)

- [x] User can edit the details of a to-do item

- [x] Every To-do item has a description and priority

- [x] Necessary run common commands like build, development should be in package.json inside `scripts'

      ex  
      "scripts": {
        "dev": "webpack --mode development",
        "build": "webpack --mode production",
        "start": "webpack-dev-server --mode development --open" // optional
      },

- [x] Use arrow functions instead of the function keyword


<hr>

## OPTIONAL

- [ ] Kindly follow the naming conventions of the file names

- [ ] the forms used to create a project or a todo should have validations, you should not be able to create an empty project/todo

Please, make the REQUIRED changes and submit your request again! :grinning:
Good luck. 

> ps: . If you have any questions about my review or need help, you can message me on slack ( @mkrtichsargsyan )

File: src/modules/frontend/_bar.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  }
+
+  list.map(
+    (item) => section.appendChild(itemWidget(parent, item, memory)),
+  );
+
+  parent.appendChild(section);
+
+  return section;
+}
```
Comment (MkrtichSargsyan) [inline src/modules/frontend/_bar.js]: use arrow functions instead of the function keyword

File: src/modules/frontend/_bar.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  if (list.length < 1) { return section; }
+
+  list.map(
+    (task) => section.appendChild(taskWidget(parent, task, memProject)),
+  );
+
+  parent.appendChild(section);
+
+  return section;
+}
```
Comment (MkrtichSargsyan) [inline src/modules/frontend/_bar.js]: Use  arrow functions instead of the function keyword

File: webpack.config.js [hunk 3]
```diff
@@ -4,63 +4,59 @@ const path = require('path');
...
+    host: 'localhost',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+
+    // Add your plugins here
+    // Learn more about plugins from https://webpack.js.org/configuration/plugins/
+  ],
```
Comment (MkrtichSargsyan) [inline webpack.config.js]: Remove the comments

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

### Happy Coding ☺️

## If you have any questions you can comment here in this PR or can always connect with me on slack ( @Sajjad Ahmad ).

**Review by taiwo2 — APPROVED** [review]
# STATUS:  APPROVED 🎉 
Thanks for implementing the changes
### WELDONE GUYS 👍🏻

**Pull request merged**